package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "node_modules/.bin/mocha && ./node_modules/.bin/eslint src/*.js test/**/*.js",
     "integration-tests": "bin/run-integration-tests.sh",
+    "stop-api": ".vinyldns/bin/remove-vinyl-containers.sh",
```

Comment:
Since `integration-tests` should kill the containers, this isn't needed for the tests; however, it would be nice to have this and also include `start-api` (which would reference `.vinyldns/bin/docker-up-vinyldns.sh  --api-only  --version 0.8.0`).

Reply:
I like that. And to put a finer point on it, this is here as a convenience for scenarios when `bin/run-integration-tests.sh` fails, in effect leaving a running VinylDNS API.